PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 <!--
```

Comment:
It says **`Added {Algorithm name} [{Language}]`** in last point. Please change that.